install.sh

```diff
@@ -1,26 +1,35 @@
 #!/bin/bash
 # Installs JUUL Labs' IntelliJ configs into your user configs.
 
+# Script adapted from:
+# https://github.com/square/java-code-styles/blob/5d09b48d82fe80403fb49ffdb28e1161d58c5ad1/install.sh
+
 echo "Installing JUUL Labs IntelliJ configs..."
 
 CONFIGS="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/configs"
 
 for i in $HOME/Library/Preferences/IntelliJIdea*  \
          $HOME/Library/Preferences/IdeaIC*        \
          $HOME/Library/Preferences/AndroidStudio* \
+         $HOME/Library/Application\ Support/JetBrains/IntelliJIdea* \
          $HOME/.IntelliJIdea*/config              \
          $HOME/.IdeaIC*/config                    \
-         $HOME/.AndroidStudio*/config
+         $HOME/.AndroidStudio*/config \
+         $HOME/Library/Application\ Support/Google/AndroidStudio* \
+         $HOME/Library/Application\ Support/JetBrains/IdeaIC*
```

Comment:
Just like language version and Gradle version, we should probably standardize the version of Android Studio or IntelliJ we use across our projects (unless we do already, but this script seems like we don't). This list of directories wouldn't have to be so large if we knew that everyone was running a modern version of either IDE. 

For example, I have both IDEs up to date and would only need this script to look through `$HOME/Library/Application\ Support/JetBrains/IdeaIC*` and `$HOME/Library/Preferences/AndroidStudio*`.

Reply:
We do not, nor do I think we need to strictly enforce it (recommended upgrades might be a good approach). Unless we find a significant issue with different team members running various IDE versions (I have not seen an issue with this yet though).

With IntelliJ, nearly any version will do just fine. You'll miss out on some nice new features if you're on an older version; but it can use the same compiler that the rest of the team is using.

With Android Studio, there is a tighter coupling with the Android gradle plugin; but that forces people to upgrade Android Studio when we upgrade it in our project `build.gradle`.

Being that paths of older IDEs don't change, I don't see any reason to drop support for installing these configurations into those older versions. ¯\\_(ツ)\_/¯